jhipster-6/bookstore-monolith/pom.xml

```diff
@@ -1125,6 +1125,8 @@
     <properties>
         <!-- Build properties -->
         <maven.version>3.0.0</maven.version>
```

Comment:
Is explicitly setting these needed?
They should be inherited from the parent pom.
Try to remove these and check that module is build on JDK 17

Reply:
I didn't touch the old settings. I will remove and try to rebuild